api/src/main/java/org/apache/iceberg/types/TypeUtil.java

```diff
@@ -118,6 +119,10 @@ public static Set<Integer> getProjectedIds(Schema schema) {
     return ImmutableSet.copyOf(getIdsInternal(schema.asStruct(), true));
   }
 
+  public static List<String> getProjectedFieldNames(Schema schema, Iterable<Integer> projectedIds) {
```

Comment:
Style: use of `get` in a method name is discouraged. It's usually not helping or could be replaced with a more specific verb like `find` or `build`. Here, I'd just omit it.